src/main/java/org/tron/core/net/node/NodeImpl.java

```diff
@@ -470,7 +473,8 @@ private void consumerAdvObjToFetch() {
     advObjToFetch.values().stream().sorted(PriorItem::compareTo).forEach(idToFetch -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Specify a type for: 'idToFetch' ([squid:S2211](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2211))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-0olHK86Hco_1sx&open=AWalU-0olHK86Hco_1sx)

[ISSUE_KEY]: AWalU-0olHK86Hco_1sx